documentation/0.13.0/html/developer-guide/toc.html

```diff
@@ -0,0 +1,19 @@
+{% raw %}
```

Comment:
The content.html and toc.html also get generated by the maven build. So jekyll's asciidoc plugin is no longer used (except for the old docs which I've not attempted to convert in this PR).